js/damage.js

```diff
@@ -120,9 +120,13 @@ function getDamageResult(attacker, defender, move, field) {
     if (typeEffectiveness === 0) {
         return {"damage":[0], "description":buildDescription(description)};
     }
-    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200))) {
+    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200) || field.isGravity)) {
         return {"damage":[0], "description":buildDescription(description)};
     }
+    if (move.name === "Synchronoise" &&
```

Comment:
Actually, what you did isn't fine, specifically because `type2` might be blank. If both `type2` are blank then it doesn't pass. I'll fix it later.